### ISSUE TITLE
Allow specifiying mongodb find options

### ISSUE THREAD

**Issue by Timeraa** [GraphQLGuide/apollo-datasource-mongodb#85]
When the DataLoader runs the mongodb query it should allow the user to specify some options like `projection` as you don't need to cache/fetch fields that you don't use in your queries.

**Comment by lolo-io** [comment]
I agree. Unfortunately the lack of options makes this datasource implementation too limited for my app.
